lca_activity_browser/app/controller.py

```diff
@@ -174,28 +185,24 @@ def add_database(self):
             "Name of new database:" + " " * 25
         )
         if name:
-            bw.Database(name).register()
-            signals.databases_changed.emit()
-            signals.database_selected.emit(name)
+            if name not in bw.databases:
+                bw.Database(name).register()
+                signals.databases_changed.emit()
+                signals.database_selected.emit(name)
+            else:
+                self.window.info("A database with this name already exists.")
 
-    def copy_database(self):
-        name = self.window.right_panel.inventory_tab.databases.currentItem().db_name
+    def copy_database(self, name):
         new_name = self.window.dialog(
             "Copy {}".format(name),
             "Name of new database:" + " " * 25)
-        if new_name:
```

Comment:
This check is required, if the user closes the dialog, the dialog returns and empty string and the new database will be called  `''`. It also leads to and indentation error, see the checks on travis. This probably went wrong when fixing the merge conflicts, because of my recent changes to this part of the code.